rules/windows/command_and_control_tool_transfer_via_curl.toml

```diff
@@ -85,9 +83,18 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and user.id != "S-1-5-18" and
```

Comment:
why not use `not (?process.Ext.token.integrity_level_name : "System" or ?winlog.event_data.IntegrityLevel : "System")` ? integrity level is populated for sysmon ?

![image](https://github.com/user-attachments/assets/5781976e-51f4-41ae-a9c4-6ac68a2d1dc3)

Reply:
We could do that, but that way, we need to drop the support for the other data sources that don't support it: Endgame, CS, S1 & Windows Logs